src/main/java/dev/hephaestus/glowcase/client/gui/screen/ingame/HyperlinkBlockEditScreen.java

```diff
@@ -24,10 +25,17 @@ public void init() {
 
 		if (this.client == null) return;
 
-		this.urlEntryWidget = new TextFieldWidget(this.client.textRenderer, width / 10, height / 2 - 10, 8 * width / 10, 20, Text.empty());
-		this.urlEntryWidget.setText(this.hyperlinkBlockEntity.getUrl());
+		this.titleEntryWidget = new TextFieldWidget(this.client.textRenderer, width / 10, height / 2 - 30, 8 * width / 10, 20, Text.empty());
+		this.titleEntryWidget.setMaxLength(Integer.MAX_VALUE);
+		this.titleEntryWidget.setText(this.hyperlinkBlockEntity.getTitle());
+		this.titleEntryWidget.setPlaceholder(Text.translatable("gui.glowcase.title"));
+
+		this.urlEntryWidget = new TextFieldWidget(this.client.textRenderer, width / 10, height / 2 + 10, 8 * width / 10, 20, Text.empty());
 		this.urlEntryWidget.setMaxLength(Integer.MAX_VALUE);
```

Comment:
Same as above

Reply:
done up in c6f928d because i'd like to clean out our PRs today